docs/api/McMaster.Extensions.CommandLineUtils.Abstractions.CommandLineContext.html

```diff
@@ -88,44 +88,20 @@ <h5>Inheritance</h5>
     <div class="level0"><span class="xref">System.Object</span></div>
     <div class="level1"><span class="xref">CommandLineContext</span></div>
   </div>
-  <div class="inheritedMembers">
-    <h5>Inherited Members</h5>
-    <div>
-      <span class="xref">System.Object.Equals(System.Object)</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.Equals(System.Object, System.Object)</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.GetHashCode()</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.GetType()</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.MemberwiseClone()</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.ReferenceEquals(System.Object, System.Object)</span>
-    </div>
-    <div>
-      <span class="xref">System.Object.ToString()</span>
-    </div>
-  </div>
   <h6><strong>Namespace</strong>: <a class="xref" href="McMaster.Extensions.CommandLineUtils.Abstractions.html">McMaster.Extensions.CommandLineUtils.Abstractions</a></h6>
   <h6><strong>Assembly</strong>: McMaster.Extensions.CommandLineUtils.dll</h6>
   <h5 id="McMaster_Extensions_CommandLineUtils_Abstractions_CommandLineContext_syntax">Syntax</h5>
   <div class="codewrapper">
-    <pre><code class="lang-csharp hljs">public abstract class CommandLineContext</code></pre>
+    <pre><code class="lang-csharp hljs">public abstract class CommandLineContext : object</code></pre>
   </div>
   <h3 id="properties">Properties
   </h3>
   <span class="small pull-right mobile-hide">
     <span class="divider">|</span>
-    <a href="https://github.com/natemcmaster/CommandLineUtils/new/master/apiSpec/new?filename=McMaster_Extensions_CommandLineUtils_Abstractions_CommandLineContext_Arguments.md&amp;value=---%0Auid%3A%20McMaster.Extensions.CommandLineUtils.Abstractions.CommandLineContext.Arguments%0Asummary%3A%20'*You%20can%20override%20summary%20for%20the%20API%20here%20using%20*MARKDOWN*%20syntax'%0A---%0A%0A*Please%20type%20below%20more%20information%20about%20this%20API%3A*%0A%0A">Improve this Doc</a>
+    <a href="https://github.com/jerriep/CommandLineUtils/new/docs/apiSpec/new?filename=McMaster_Extensions_CommandLineUtils_Abstractions_CommandLineContext_Arguments.md&amp;value=---%0Auid%3A%20McMaster.Extensions.CommandLineUtils.Abstractions.CommandLineContext.Arguments%0Asummary%3A%20'*You%20can%20override%20summary%20for%20the%20API%20here%20using%20*MARKDOWN*%20syntax'%0A---%0A%0A*Please%20type%20below%20more%20information%20about%20this%20API%3A*%0A%0A">Improve this Doc</a>
```

Comment:
Probably need to regenerate this using a different origin and branch setting

Reply:
Yep, see my other comment below. I will create a new PR once we are done here to exclude all those docfx generated docs.